src/components/SecurityNotifier.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <section class="security-notify">
+    <div v-if="!securityEnabled" class="alert alert-warning">
+      <p><b>This data source is missing security rules.</b></p>
+      <p>Configure security rules so the app can access the data</p>
+      <div @click="() => { this.$emit('updateSecurityRules') }" class="btn btn-primary security-btn">Configure security rules</div>
```

Comment:
Use function and change class name.

```suggestion
      <div @click="foo" class="btn btn-primary btn-security">Configure security rules</div>
```